packages/client/src/schedule-client.ts

```diff
@@ -377,24 +374,22 @@ export class ScheduleClient extends BaseClient {
       );
 
       for (const raw of response.schedules ?? []) {
-        if (!raw.info?.spec) continue;
-
         yield <ScheduleSummary>{
           scheduleId: raw.scheduleId,
 
-          spec: decodeScheduleSpec(raw.info.spec),
-          action: {
+          spec: decodeScheduleSpec(raw.info?.spec ?? {}),
+          action: raw.info?.workflowType?.name && {
```

Comment:
We need to fix this on the server side and make the action non-nullable.

Reply:
I agree, but in the mean time, I think we're better provide compatibility with 1.18.x and 1.19.x, which are people are currently using. Making optional properties required on that interface at a later point will be non breaking, as that interface is meant to be a return value from our API.